app/views/jobseekers/job_applications/review/_professional_status.html.slim

```diff
@@ -9,7 +9,7 @@
     = govuk_summary_list do |component|
       - component.slot(:row,
         key: t("helpers.legend.jobseekers_job_application_professional_status_form.qualified_teacher_status"),
-        value: safe_join([tag.div(t("helpers.label.jobseekers_job_application_professional_status_form.qualified_teacher_status_options.#{job_application.qualified_teacher_status}"), class: "govuk-body"),
+        value: safe_join([tag.div(job_application.readable_qualified_teacher_status, class: "govuk-body"),
```

Comment:
Should we be passing anything to `value` at all if qualified_teacher_status is nil?

Reply:
We should display the row, but I suppose I should pass a nil to value.